### PR TITLE
feat(filter): show workitem types filters in alphabetical order

### PR DESCRIPTION
**Pull request by debloper**
### [ ] What does this PR do? _[Mandatory]_
$title

### [ ] What issue/task does this PR references? _[Mandatory]_
Closes openshiftio/openshift.io#2604

### [ ] Are the tests Included? _[Mandatory]_
N/A

#### [ ] Is the documentation Included?
N/A

#### [ ] Are the Release Notes included?
N/A

#### [ ] Is/Are the @mention(s) to reviewer(s) included
Yes

### CODE REVIEW

**Comment by rgarg1** [comment]
@debloper Deployment is failing. How do I validate the fix?

**Comment by rgarg1** [comment]
@debloper LGTM.

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts [hunk 1]
```diff
@@ -391,7 +391,15 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
         datasource: this.workItemTypeData,
         datamap: (witypes: WorkItemTypeUI[]) => {
           return {
-            queries: witypes.map(witype => {return {id: witype.id, value: witype.name, iconStyleClass: witype.icon}}),
+            queries: witypes.sort((a, b) => {
+              return (a.name > b.name ? 1 : 0)
```
Comment (vikram-raj) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: A semicolon is missing.

Comment (debloper) [reply src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: A byte saved... hallelujah!!!

Semicolon in a single line return statement has no programmatic impact.
However, I'll update this commit to match with the linter expectations.

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @debloper for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by alien-ike** [comment]
Hey @debloper! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer, but not pull request creator for this command to take an effect.

**Comment by debloper** [comment]
@nimishamukherjee you have to add `/ok-without-tests` as it doesn't accept the command from admins who raised the PR (me).

**Pull request merged**